### ISSUE TITLE
iOS custom marker seems zoomed in

### ISSUE THREAD

**Issue by StrahinjaCav** [dapriett/nativescript-google-maps-sdk#278]
iOS custom marker feels zoomed in, i tried resizing icons in the backend and that brought them to a normal size but now they are blurred because they are too small.
Did anyone have this issue, it feels weird nobody mentioned it in these 200 issues

**Comment by NicoDino** [comment]
Hi @StrahinjaCav , I'm having the same issue with iOS. 
The following code is working ok for Android, the images resizing is automatic when you zoom in or out. 
We can't get it working with iOS tough, could you solve it?

```
    let imageSourceOrigen = new ImageSource();
        await imageSourceOrigen.fromFile('~/assets/images/marker.png');
        let iconOrigen = new Image();
        iconOrigen.imageSource = imageSourceOrigen;
        this.markerOrigen.icon = iconOrigen;
```

**Comment by NickAtGit** [comment]
Yes I'm seeing the same. The markers I usually use for a real iOS project are mostly around 50x50 pixel. Those marker seem right on iOS but on Android they are 5 times smaller than on Android. To me the solution can't be to store icons in different sizes but I haven't found a neat solution so far... :/

**Comment by csi-stasik** [comment]
As a workaround you can use this:

```
const marker: Marker = new Marker();
marker.icon = "myImageName";

const scaledIcon = UIImage.alloc().initWithCGImageScaleOrientation(marker.ios.icon.CGImage, 3, 0);
marker.ios.icon = scaledIcon;
```
